examples/Dicom/src/parseDicomFiles.js

```diff
@@ -0,0 +1,194 @@
+import PromiseFileReader from 'promise-file-reader'
+import dicomParser from 'dicom-parser'
+
+import "regenerator-runtime/runtime";
+
+const DICOM_DICTIONARY = {
+  patientId: 'x00100020',
+  patientName: 'x00100010',
+  patientDateOfBirth: 'x00100030',
+  patientSex: 'x00100040',
+  studyID: 'x00200010',
+  studyUID: 'x0020000d',
+  studyDate: 'x00080020',
+  studyTime: 'x00080030',
+  studyAccessionNumber: 'x00080050',
+  studyDescription: 'x00081030',
+  seriesNumber: 'x00200011',
+  seriesUid: 'x0020000e',
+  seriesDate: 'x00080021',
+  seriesTime: 'x00080031',
+  seriesModality: 'x00080060',
+  seriesDescription: 'x0008103e',
+  seriesProtocolName: 'x00181030',
+  seriesBodyPart: 'x00180015',
+}
+
+class DICOMEntity {
+  constructor() {
+    this.checkTagsValidity() // could it be a static assertion instead?
+  }
+
+  checkTagsValidity() {
+    const name = this.constructor.name
+    const tags = this.constructor.tags
+    const primaryTag = this.constructor.primaryTag
+    console.assert(
+      tags.includes(primaryTag),
+      `The primary tag of the ${name} class ("${primaryTag}") is not included in its list of tags ([${tags}]).`
+      )
+    tags.forEach((tag) => {
+      console.assert(
+        tag in DICOM_DICTIONARY,
+        `The tag "${tag}" associated with the ${name} class is not defined in DICOM_DICTIONARY.`
+        )
+    })
+  }
+
+  extractTags(dicomMetaData) {
+    const tags = this.constructor.tags
+    const primaryTag = this.constructor.primaryTag
+    tags.forEach((tag) => {
+      const value = dicomMetaData.string(DICOM_DICTIONARY[tag])
+      if (this[tag] === undefined) {
+        this[tag] = value
+      } else if (value != undefined) {
```

Comment:
This should be fine since you're comparing with undefined, but FYI linters will likely want to see `!==` instead of `!=`.

Reply:
I would go further and recommend that if you're attempting to compare with undefined, definitely use strict equality comparison, and if you only care about truthiness, be explicit and just do `if (value)`.